go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/terraform-providers/terraform-provider-tfe
 
 require (
-	github.com/hashicorp/go-hclog v0.0.0-20190109152822-4783caec6f2e // indirect
+	github.com/google/btree v1.0.0 // indirect
 	github.com/hashicorp/go-tfe v0.4.0
 	github.com/hashicorp/go-version v1.2.0
 	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce
 	github.com/hashicorp/terraform v0.12.0
```

Comment:
After migrating to the SDK, the provider should no longer depend on `github.com/hashicorp/terraform`, so this line should not appear in `go.mod`. 

I suspect Go may have added this back in after you ran `tf-sdk-migrator migrate` due to running some other `go` commands (perhaps without being in module mode). You may need to go back to https://github.com/terraform-providers/terraform-provider-tfe/pull/124/commits/130cdf37ba01b0d105775ca046d0769e800f47c9 and do the migration again, making sure to run `go mod vendor` directly afterwards.